src/cc/api/BPF.h

```diff
@@ -287,6 +287,8 @@ class USDT {
                << usdt.probe_func_;
   }
 
+  int set_probe_matching_kludge(uint8_t kludge);
```

Comment:
this can be removed?

Reply:
addressed this on your other `set_probe_matching_kludge` comment